ansible_collections/arista/avd/molecule/evpn_underlay_ebgp_overlay_ebgp/documentation/devices/DC1-SVC3A.md

```diff
@@ -1286,7 +1288,7 @@ ip route vrf MGMT 0.0.0.0/0 192.168.200.5
 | Tenant_A_APP_Zone | 192.168.255.12:12 | connected |
 | Tenant_A_DB_Zone | 192.168.255.12:13 | connected |
 | Tenant_A_OP_Zone | 192.168.255.12:10 | connected |
-| Tenant_A_WAN_Zone | 192.168.255.12:14 | connected |
+| Tenant_A_WAN_Zone | 192.168.255.12:14 | connected<br>static |
```

Comment:
Why is there a "\<br>" here? Otherwise it looks very good.

Reply:
In some doc templates we have `<br>` to list multiple items in a single field. Other templates use `, `.
I have not changed anything in this regard, but since I add static routes to the VRF, the templates auto-detect that we need to redistribute static as well.